components/Currency/CurrencyIcon/CurrencyIcon.tsx

```diff
@@ -79,11 +94,77 @@ const CurrencyIcon: FC<CurrencyIconProps> = ({ currencyKey, type, isDeprecated,
 
 	if (!firstFallbackError) {
 		switch (currencyKey) {
-			case CRYPTO_CURRENCY_MAP.ETH: {
-				return <ETHIcon {...props} />;
+			case FuturesMarketKey.sETH: {
```

Comment:
One other option here is we could create a mapping to get the image src and then just return the one Image tag like:
```
const synthIcons = {
  sETH: ETHIcon,
  sUNI: UNIIcon,
  ...
}

const src = synthIcons[currencyKey]
return src ? <Image src={src} layout="raw" {...props} /> : <TheDefaultImage />
```

Reply:
Good idea. Let me make a change.

Reply:
@avclarke Change has been made. Please have a re-review. Thanks.